star_tides/services/mongo/config.ini

```diff
@@ -0,0 +1,7 @@
+[mongo]
+host = mongodb_migrations
+port = 27017
+database = test
```

Comment:
Do any of these need to be updated or parameterized?

Reply:
We aren't using mongo migrations right now. This was a test to try and get them working.

Reply:
Cool - yeah let's delete it for now. We might need to use it in the future but let's not complicate things now.